tests/conftest.py

```diff
@@ -303,6 +313,7 @@ def _slurm_not_installed():
 def _subprocess_setup(request, cfg=None, uwsgi=False):
     """setup the supervisor"""
     import os
+    import sirepo.const
```

Comment:
from sirepo import const when it is inside a function

Reply:
[308777d](https://github.com/radiasoft/sirepo/pull/4846/commits/308777d2447203787dc3df0950124714dd1d90f7)